lib/ruby_lsp/tapioca/server_addon.rb

```diff
@@ -20,7 +20,8 @@ def execute(request, params)
       private
 
       def dsl(params)
-        # TODO
+        load("tapioca/cli.rb") # Reload the CLI to reset thor defaults between requests
```

Comment:
Ugh, this is a bit odd. I think this is fine for the first iteration, but it would be nice to invest in separating the CLI from the add-on interface.